test/evergreen.yml

```diff
@@ -371,6 +382,7 @@ functions:
             ${python_binary|python3} ../test/evergreen/print_stack_trace.py
           fi
           exit 0
+    - *upload_stack_trace
```

Comment:
Could we do that for the full dump stack traces function, instead?

Reply:
Sorry I didn't get that.
We are missing the upload of the stack trace file here hence I added this call here. What do you mean by the `full dump stack traces function`? The one called in the post tasks?

Reply:
Yes, can't we make an alias of the dump stack trace function? Then use that here?

Reply:
Hmm, it would be a better solution indeed.
The thing is that we are relying on `exit_code` just before calling the script. Now I am thinking about it, we could just call it all the time?

Reply:
Yea, sounds good to me.